desktop/qt6ct/qt6ct.SlackBuild

```diff
@@ -31,13 +31,15 @@ TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
 
 if [ -z "$ARCH" ]; then
-  case "$( uname -m )" in
-    i?86) ARCH=i586 ;;
-    arm*) ARCH=arm ;;
-       *) ARCH=$( uname -m ) ;;
-  esac
+  ARCH="$( uname -m )"
 fi
 
+case "$ARCH" in
```

Comment:
Patrick supports qt6 on -current (Steven could probably pull in Pat's patches to his SBo build too), so I wonder if we should not force this to fail here?